quickfixj-core/src/main/java/quickfix/mina/ssl/SSLFilter.java

```diff
@@ -49,4 +59,26 @@ public void setEnabledCipherSuites(String[] cipherSuites) {
     public void setCipherSuites(String[] cipherSuites) {
         super.setEnabledCipherSuites(cipherSuites);
     }
+
+    @Override
+    public void onPreAdd(IoFilterChain parent, String name, NextFilter nextFilter)
+        throws SSLException {
+
+        if (useSNI) {
+            IoSession session = parent.getSession();
+            SocketAddress remoteAddress = session.getRemoteAddress();
+
+            if (remoteAddress instanceof InetSocketAddress) {
+                // activate the SNI support in the JSSE SSLEngine
+                log.info("activating TLS SNI support for peer address: " + remoteAddress);

```

Comment:
No need for +, you can use the argument

Reply:
That doesn't seem to be the style established in the project (or, I don't understand what you mean by "argument")
![image](https://user-images.githubusercontent.com/588573/77955661-3f148300-72d9-11ea-8dc6-3a19998eacf8.png)

Reply:
I think it is mixed, depends which class you look at e.g. `quickfix.mina.SingleThreadedEventHandlingStrategy#blockInThread`. If you keep + I will not complain.

Reply:
Changed.